capstone_project/src/main/java/com/google/sps/data/Resource.java

```diff
@@ -0,0 +1,75 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import java.util.HashMap;
+import java.util.Map;
+
+/** Associates a category, from the Natural Language API taxonomy, to a link to a resource */
+public class Resource {
+  private final DatastoreService datastore;
+  private final String category;
+  private final String resource;
+
+  public Resource() {
+    datastore = DatastoreServiceFactory.getDatastoreService();
+    this.category = "";
+    this.resource = "";
+
+    addPreexistingResources();
```

Comment:
this should probably be called explicitly from somewhere else since it's confusing to see an object get constructed but not assigned to anything